import-service/src/functions/catalogBatchProcess/handler.ts

```diff
@@ -0,0 +1,66 @@
+import { SNSClient, PublishCommand } from '@aws-sdk/client-sns';
+
+import { formatJSONResponse } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import { Handler, SQSEvent } from 'aws-lambda';
+import createError from 'http-errors';
+
+const getValues = (fieldsNum: number, rowsNum: number) => {
+  let values = '';
+  for (let i = 1; i <= rowsNum; i++) {
+    values += '(';
+    for (let j = (i - 1) * fieldsNum + 1; j <= i * fieldsNum; j++) {
+      values += `$${j}${j < i * fieldsNum ? ',' : ''}`
+    }
+    values += `)${i < rowsNum ? ',' : ''}`;
+  }
+  return values;
+};
```

Comment:
Wow, looks like overengineered solution. It's better to use [pg-format](https://github.com/datalanche/node-pg-format) package. Then it could be just
```
import format from "pg-format";

const values = products.map((product) => {
  return [product.title, product.description, product.price];
});
const query = format(
    "insert into products (title, description, price) values %L returning id",
     values
   ),
};
```

But, I believe, if you hardcode `$1, $2, ...` it will be more readable than what you have now. If you hardcoded columns names (`title, description, price`) why didn't you hardcode values for them?

Reply:
Yes, you're absolutely right! I found pg-format yesterday and planned to use it, after some other changes